lizmap/modules/lizmap/classes/qgisVectorLayerDatasource.class.php

```diff
@@ -96,7 +97,7 @@ private function getDatasourceParameterSql($param)
             $backSlashedQuoteReplacement = '@@@LIZMAP@@@';
             preg_match(
                 '#'.$regex.'#s',
-                str_replace('\\"', $backSlashedQuoteReplacement, $this->datasource),
+                str_replace('\"', $backSlashedQuoteReplacement, $this->datasource),
```

Comment:
We will check if `getDatasourceParameterSql` is covered by a unit test.

Reply:
It's a commit from @mdouchin 5c81b2bc1097136aef35757c31b8182605c4faf0

> We need to replace the \" in the datasource to avoid issues for complex sub-queries in table=()

Notice the `\"` in the comment. Will look

Reply:
It seems well covered by unit tests : 

> AS \\\"population\\\" FROM gobs.observation

> table=\"referentiel\".\"lieux\"

So I would say it's indeed OK.